From 83ceb1a4b07d1e2eda0d25163d9e4c60eaa8a215 Mon Sep 17 00:00:00 2001
From: "Kelvin M. Klann" <kmk3.code@protonmail.com>
Date: Mon, 22 Sep 2025 15:42:20 -0300
Subject: [PATCH] profiles: wusc: add /usr/share/glycin-loaders

This is apparently needed by glycin/gdk-pixbuf2, which is used by many
programs, such as Firefox and GIMP.

Relates to #6906.

Reported-by: @myrslint
Suggsted-by: @myrslint
---
 etc/inc/whitelist-usr-share-common.inc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/etc/inc/whitelist-usr-share-common.inc b/etc/inc/whitelist-usr-share-common.inc
index ed5367208e..5a28cdac62 100644
--- a/etc/inc/whitelist-usr-share-common.inc
+++ b/etc/inc/whitelist-usr-share-common.inc
@@ -23,6 +23,7 @@ whitelist /usr/share/gir-1.0
 whitelist /usr/share/gjs-1.0
 whitelist /usr/share/glib-2.0
 whitelist /usr/share/glvnd
+whitelist /usr/share/glycin-loaders
 whitelist /usr/share/gtk-2.0
 whitelist /usr/share/gtk-3.0
 whitelist /usr/share/gtk-engines
